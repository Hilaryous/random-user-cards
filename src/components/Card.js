// @flow
import styled from 'react-emotion'

const Card = styled('li')({
  display: 'block',
  backgroundColor: 'white',
  boxShadow: '0 0 6px 0 rgba(0, 0, 0, 0.2)',
  padding: 20,
})

export default Card
