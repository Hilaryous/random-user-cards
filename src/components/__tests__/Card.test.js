/* eslint-disable function-paren-newline */
import React from 'react'
import renderer from 'react-test-renderer'
import Card from '../Card'

describe('Card', () => {
  it('shows the correct snapshot tree for the Card component', () => {
    const tree = renderer.create(
      <Card />,
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
