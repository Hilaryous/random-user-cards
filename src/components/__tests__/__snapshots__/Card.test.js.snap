// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card shows the correct snapshot tree for the Card component 1`] = `
<li
  className="css-ouze9h"
/>
`;
